ONSdigital/eq-author-app
# Relative NextPage routing destination now displays as 'Next page'

### What is the context of this PR?

When adding a routing rule to a page for the first time, it creates a blank rule with the destination being the next page in the questionnaire. However, this is displayed to the user as the name of the question. The purpose of this task was to change the text displayed to the user to "Next page".

### How to review 


* Create a new questionnaire with a routable answer as question one and any other type as question two.
* Create a new routing rule on question one; the destination fields should show "Next page" instead of the destination question name. 
* Ensure the routing still works.

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
